pyproject.toml

```diff
@@ -52,7 +46,7 @@ skip_glob = [".direnv", "venv", ".venv"]
 use_parentheses = true
 
 [tool.pytest.ini_options]
-addopts = "--disable-network --tb=native --no-migrations --ignore=./release-hatch"
+addopts = "--disable-network --tb=native --no-migrations --ignore=./release-hatch -n auto"
```

Comment:
I think we shouldn't switch this on by default. This so we can always run normally to debug parallel test issues.

We could have a `just test` which is then parallel by default, and maybe a `just test-serial` for those cases?

Also, locally I don't see any significant benefit from >4 workers, so maybe add `--maxprocesses=4` here instead?
So we can run parallel with `just test -n N`